cardano-sl.cabal

```diff
@@ -115,8 +113,26 @@ library
                         Pos.Util.UserSecret
 
                         -- Wallets
+                        Pos.Wallet
                         Pos.Wallet.KeyStorage
+                        Pos.Wallet.SscType
+                        Pos.Wallet.WalletMode
+                        Pos.Wallet.Web.ClientTypes
+                        Pos.Wallet.Web.Error
+                        Pos.Wallet.Web.Error.Types
+                        Pos.Wallet.Web.Error.Util
                         Pos.Wallet.Web.Secret
+                        Pos.Wallet.Web.State
+                        Pos.Wallet.Web.State.Acidic
+                        Pos.Wallet.Web.State.State
+                        Pos.Wallet.Web.State.Storage
+                        Pos.Wallet.Web.Util
```

Comment:
Why are these `Pos.Wallet.*` modules still in `cardano-sl`? Does something else (not wallet) depend on it?